pom.xml

```diff
@@ -134,11 +146,33 @@
                 <groupId>org.jenkins-ci.tools</groupId>
                 <artifactId>maven-hpi-plugin</artifactId>
                 <configuration>
-                    <pluginFirstClassLoader>true</pluginFirstClassLoader>
                     <maskClasses>
-                        com.microsoft.azure.management.
                         com.google.common.
                         com.fasterxml.jackson.
+                        org.apache.commons.lang3.
+
+                        com.microsoft.azure.AzureAsyncOperation
+                        com.microsoft.azure.AzureClient
+                        com.microsoft.azure.AzureEnvironment
+                        com.microsoft.azure.AzureResponseBuilder
+                        com.microsoft.azure.AzureServiceClient
+                        com.microsoft.azure.AzureServiceFuture
+                        com.microsoft.azure.CloudError
+                        com.microsoft.azure.CloudException
+                        com.microsoft.azure.ListOperationCallback
+                        com.microsoft.azure.Page
+                        com.microsoft.azure.PagedList
+                        com.microsoft.azure.PollingState
+                        com.microsoft.azure.Resource
+                        com.microsoft.azure.SubResource
+                        com.microsoft.azure.credentials.
+                        com.microsoft.azure.keyvault.
+                        com.microsoft.azure.management.
+                        com.microsoft.azure.serializer.
+                        com.microsoft.azure.storage.
+                        com.microsoft.rest.
```

Comment:
Mask all the classes in Azure SDK to resolve class loader conflict. As a result, all the Azure SDK classes will be loaded from this plugin, rather than relying on the SDK version configured in `azure-credentials`.